docs/extensions/toc.md

```diff
@@ -196,21 +196,16 @@ The following options are provided to configure the output:
 * **`separator`**:
     Word separator. Character which replaces white space in id. Defaults to "`-`".
 
-* **`toc_height`**
-    Define the highest section level "n" (`<hn>` to `<h6>`, where `1 <= n <= 6`)
-    to include in the Table of Contents. Section levels above are omitted.
-    Defaults to `1`.
-
-    When used with conjunction with `baselevel`, this parameter will not
-    take the fitted hierarchy from `baselevel` into account.
-    If you set `baselevel` to 3 and `toc_height` to 2, the *first* headline
-    will be `<h3>` and so still included in the Table of Contents. To exclude
-    this first level, you have to set `toc_height` to 4.
-
 * **`toc_depth`**
-    Define up to which section level "n" (`<h1>` to `<hn>`, where `1 <= n <= 6`)
-    to include in the Table of Contents. Defaults to `6`.
+    Define the range of section levels to include in the Table of Contents.
+    A single integer (b) defines the bottom section level (<h1>..<hb>) only.
+    Two digits seperated by a hyphen in between (```2-5```), define the
```

Comment:
I'm assuming you are trying to indicate that the range should be wrapped in quotes. However, triple backpacks won't give you that. You need to actually use quotes: `"2-5"`. And as a string is required, the docs should probably mention that the range needs to be a string consisting of "Two digits separated by a hyphen".

Also it is spelled "sep**a**rated", not "seperated."